src/AppInstallerCLITests/WorkFlow.cpp

```diff
@@ -210,6 +210,21 @@ namespace
                         PackageMatchFilter(PackageMatchField::Id, MatchType::Exact, "AppInstallerCliTest.TestExeInstaller")));
             }
 
+            if (input == "TestExeInstallerWithDifferentInstalledType")

```

Comment:
I'm not a fan of how long this function is getting. Many tests seem to use only a single case of all this and most are used in a single test, so we probably could add a constructor that took the manifests and always returned that. It would also put the search results closer to where it's used. Putting this here to see what others think of doing that..

Reply:
Agree, making it easy to create specific new cases rather than extending this would be good.